client/views/account/AccountProfileForm.js

```diff
@@ -150,9 +150,8 @@ function AccountProfileForm({ values, handlers, user, settings, onSaveStateChang
 				: t('Max_length_is', STATUS_TEXT_MAX_LENGTH),
 		[statusText, t],
 	);
-	const {
-		emails: [{ verified = false } = { verified: false }],
-	} = user;
+
+	const verified = (user.emails && user.emails.length && user.emails[0].verified) || false;
```

Comment:
Why not just adding a default value to the destructuring?
```
const { emails: [{ verified = false } = { verified: false }] } = user || {}
```

Reply:
@KevLehman Thanks for your review 😄 
Adding default here won't work here, As `emails` is an array and we need to get `verified` from the first email.
The validation I have added is already there at most of the other places in the code while using `emails`.

<img width="1791" alt="Screenshot 2021-06-23 at 10 15 28 AM" src="https://user-images.githubusercontent.com/18264684/123037456-55da5480-d40c-11eb-9e43-06dd820c8ba1.png">

Reply:
Ah, that's right, you're handling for null here. In that case, using the `?.` would look cleaner. And that should also work with null :) 
```
user?.emails?.[0]?.verified || false
```